common/source/docs/common-frsky-flightdeck.rst

```diff
@@ -0,0 +1,43 @@
+.. _common-frsky-flightdeck:
```

Comment:
`a product developed to use one of the telemetry options available on ArduPilot, so not directly necessary to use the telemetry`
Just a further clarification, I know that some Open Source projects might be using the new telemetry protocol as soon as the required system base versions required become released